leptos_dom/src/node_ref.rs

```diff
@@ -146,7 +188,7 @@ impl<T: ElementDescriptor + 'static> NodeRef<T> {
     where
         T: Clone,
     {
-        self.0.update(|current| {
+        self.element.update(|current| {
             if current.is_some() {
                 crate::debug_warn!(
                     "You are setting a NodeRef that has already been filled. \
```

Comment:
You can update the warning here to something like `"You are setting the NodeRef defined at {}, which has already been filled...", self.defined_at` so it actually uses the new information.

Reply:
Thanks that look better .. pushed